docs/src/main/sphinx/release/release-474.md

```diff
@@ -0,0 +1,13 @@
+# Release 474 (21 Mar 2025)
+
+## General
+* Add `originalUser` and `authenticatedUser` as resource group selectors. ({issue}`24662`)
+* Use JDK 24 in the runtime ({issue}`23501`)
+* Fix a correctness bug in GROUP BY or DISTINCT queries with a large number of unique groups ({issue}`25381`)
+
+## Delta Lake
+* Fix failure for `MERGE` queries on [cloned](https://delta.io/blog/delta-lake-clone/) tables. ({issue}`24756`)
+
+## Iceberg connector
+* Add support for `iceberg.rest-catalog.session-timeout` and `iceberg.rest-catalog.oauth2.token-refresh-enabled` catalog config properties. ({issue}`25160`)
```

Comment:
What functionality is being added? Describe in those terms instead. Also, they appear to be two independent changes sharing one PR, so they should be listed separately.

Reply:
Broken into two parts and reworded